website/util.ts

```diff
@@ -0,0 +1,160 @@
+// Copyright 2018 Ryan Dahl <ry@tinyclouds.org>
+// All rights reserved. MIT License.
+
+import * as ts from "typescript";
+import * as types from "./types";
+
+const mapSeparator = Symbol();
+
+export class One2ManyMap<KeyType, ValueType> {
```

Comment:
Please add a `util_test.ts` for checking the proper functioning of One2ManyMap.

Reply:
done